panel/src/components/Drawers/BlockDrawer.vue

```diff
@@ -63,6 +63,12 @@ export default {
 		"show",
 		"submit",
 		"tab"
-	]
+	],
+	methods: {
+		show() {
+			this.hidden = false;
```

Comment:
Don't you get console errors with this? Usually Vue is strictly against directly mutating props.

Reply:
Hmm, no 🤷‍♂️

Reply:
I'm not expert about that but I think you're right. I have updated as your idea anyway.

Reply:
I think we should update the drawer props instead: 

```js
this.$panel.drawer.props.hidden = false;
```

Reply:
I think my last change that Nico's idea is better :)

Reply:
@afbora I wouldn't create a new data entry. It introduces just another place this needs to be updated, another place that could be false later. Better to have a single place of truth (the prop).

@bastianallgeier I also don't think directly updating the drawer prop is the best idea. We should find out why emitting the event isn't working - or why it is working but the drawer props aren't reactively updating. I think that's the actual bug.

As I think about it, it's probably the same issue as https://github.com/getkirby/kirby/issues/6301

Reply:
@distantnative I looked into it a bit. The problem seemed simple but I guess it's more difficult than I expected. Feel free to close the PR

Reply:
I think I would indeed rather keep this small bug around until we have a proper fix of the reactivity. Instead of patch-fixing only this one occurrence. What do you think @bastianallgeier ?